code/.gitignore

```diff
@@ -36,6 +36,12 @@ examples/angular-cli/addon-jest.testresults.json
 junit.xml
 .next
 
+/test-results/
+/playwright-results/
+/playwright-report/
+/playwright/.cache/
```

Comment:
Should we aim to have these files in `/scripts` instead?

Reply:
Makes sense. Where should the actual script live @ndelangen ?